src/desktop/components/fair_week_marketing/__tests__/BannerPopUp.jest.tsx

```diff
@@ -9,7 +9,7 @@ jest.mock("sharify", () => ({
       {
         slug: "ca3",
         copy: "Discover and Buy Works from Art Fairs",
-        image: "http://files.artsy.net/images/art-fair.jpg",
+        image: "https://files.artsy.net/images/art-fair.jpg",
```

Comment:
Random thing I noticed when doing this; the copy-to-clipboard button in Vanity copies an http URL instead of https.